README.md

```diff
@@ -39,37 +45,54 @@ You can contribute by proposing new goals for a focus area, or new questions for
 or new metrics for answering those questions. Or by helping to refine goals, and questions.
 
 
-## Meetings
```

Comment:
I think we need info about meetings somewhere, since this is one of the main activities of the wg.

Reply:
+1 I think we need this info. Also information about mailing list.

Reply:
I see info about meetings and mailing list below, so resolving this comments.